facdb/sql/nysomh_mentalhealth.sql

```diff
@@ -28,14 +28,7 @@ SELECT
     'NYSOMH' as overabbrev,
     NULL as capacity,
     NULL as captype,
-    (CASE
-        WHEN location LIKE '%(%'
-            THEN ST_SetSRID(ST_Point(
-                split_part(REPLACE(reverse(split_part(reverse(location),
-                            '(', 1)),')',''), ',', 2)::DOUBLE PRECISION,
-                split_part(REPLACE(reverse(split_part(reverse(location),
-                            '(', 1)),')',''), ',', 1)::DOUBLE PRECISION), 4326)
-    END) as wkb_geometry,
+    ST_AsBinary(ST_AsText(georeference)) as wkb_geometry,
```

Comment:
ran on my machine, see most of values in `wkb_geometry` in `_nysomh_mentalhealth` populated with binary values